commands/project/cloneApp.ts

```diff
@@ -153,14 +153,14 @@ exports.handler = async options => {
       }
       logger.log('');
       uiLine();
-      logger.success(i18n(`${i18nKey}.cloneStatus.success`, { location }));
+      logger.success(i18n(`${i18nKey}.cloneStatus.success`, { dest }));
       logger.log('');
       process.exit(EXIT_CODES.SUCCESS);
     }
   } catch (error) {
     trackCommandMetadataUsage(
       'clone-app',
-      { projectName: name, appId, status: 'FAILURE', error },
+      { projectName, appId, status: 'FAILURE', error },
```

Comment:
Are these properties tracked with the way that the usage tracking API is set up?

Reply:
oh good call. They probably aren't even being tracked. I can remove them

Reply:
Fixed in [6d99f67](https://github.com/HubSpot/hubspot-cli/pull/1266/commits/6d99f67a0643d7d87a8c75fc8f866891f70fcfeb)